packages/app-page-builder/src/render/plugins/elementSettings/grid/index.ts

```diff
@@ -7,7 +7,7 @@ export default {
     name: "pb-render-page-element-style-grid",
     type: "pb-render-page-element-style",
     renderStyle({ element, style }) {
-        const gridOptions = get(element, "data.settings.gridOptions");
+        const gridSettings = get(element, "data.settings.gridSettings");
```

Comment:
To make this backwards-compatible, let's `get` from both the new and the old paths:

- `data.settings.gridSettings`
- `data.settings.gridOptions`

`get` value from the first one, and if it doesn't exist, fall back to the second (old) one.


In other plugins, editor plugins, make sure you support the old data as well, otherwise we'll break existing pages.

Reply:
@Pavel910 both properties (old `gridOptions` and new `gridSettings`) were added in this pull, so looks like no need to do this.
`grip` property used for size was left as is (it was hardcoded in one of the initial pages, so I have left it unchanged).

But agree that is important thing and will try to keep an eye on such changes in future tasks.

Reply:
@neatbyte-vnobis ah ok, sorry. Too many properties to keep in mind 😅 All good then, thanks!